src/frequenz/sdk/microgrid/_power_distributing/_component_managers/_battery_manager.py

```diff
@@ -151,6 +151,10 @@ def __init__(
         self._api_power_request_timeout = api_power_request_timeout
         self._fallback_power = fallback_power
 
+        assert (
+            self._fallback_power == Power.zero()
+        ), "Non-zero fallback power is not supported for batteries."
+
```

Comment:
Maybe you could mention this in the `fallback_power` docs 4 lines above?
It only says:
```
            fallback_power: The power to assume a battery has if the battery is not
                reachable.
```

Reply:
Done.